libafl_frida/src/asan/errors.rs

```diff
@@ -418,7 +432,9 @@ impl AsanErrors {
                     writeln!(output, "allocation was zero-sized").unwrap();
                 }
 
-                if let Some(backtrace) = metadata.allocation_site_backtrace.as_mut() {
+                let mut allocation_site_backtrace = metadata.allocation_site_backtrace.clone();
```

Comment:
@s1341 can we get around this clone? Metadata needs to be borrowed in order to `resolve` the backtrace, but I'm not sure if resolving will somehow alter the state of the `error`...